src/open_inwoner/scss/components/Questionnaire/Questionnaire.scss

```diff
@@ -13,12 +13,12 @@
   }
 
   &__list-item {
-    border-bottom: var(--border-width) solid var(--color-mute);
+    border-bottom: var(--border-width) solid var(--color-gray);
```

Comment:
`--border-width` is 2px, maybe we should adhere to the card and dropdown buttons size -> 1px?
If so, let's create a variable for border size and set it to 1px, just to be reusable.

![image](https://user-images.githubusercontent.com/63058071/218483600-c9ba914e-39a0-44b9-a5cc-e1749b1e45cf.png)

Reply:
Yes, you are right, I think this should become 1px as well; i will add to it.